src/projection/projection.tsx

```diff
@@ -19,15 +19,14 @@ interface AreaRangeState {
 }
 
 const createHighChartOptions = (
-    rangeData: AreaRangeData,
+    contactData: ContactData,
+    timeSeriesData: number[],
     maxValue: number,
     type: string,
     index: number
 ): Highcharts.Options => {
-    const title =
-        rangeData.contact === "100"
-            ? "No intervention"
-            : rangeData.contact + "% contact";
+    const contact = contactData.contact.replace("data", "");
```

Comment:
we shouldn't have a contact with value `data100`. Let's just return `100`?

Reply:
oh shit i merged the backend just now. i will create another pr for this.